### PR TITLE
plugins:puppetdb_foreman - Release 0.1.3 (DEB)

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: plugins/ruby-puppetdb-foreman/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-puppetdb_foreman (0.1.3-1) stable; urgency=low
+
+  * Update release to 0.1.3
+
+ -- Daniel Lobato Garcia <elobatocs@gmail.com>  Sun, Oct 25 2015 13:40:36 +0100
```
Comment (mmoll) [inline plugins/ruby-puppetdb-foreman/debian/changelog]: the timestamp has the wrong format, it has to be `25 Oct...`

File: plugins/ruby-puppetdb-foreman/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+ruby-puppetdb_foreman (0.1.3-1) stable; urgency=low
```
Comment (mmoll) [inline plugins/ruby-puppetdb-foreman/debian/changelog]: sorry, didn't catch that in the first version... dash instead of underscore is needed here.

Comment (dLobatog) [inline plugins/ruby-puppetdb-foreman/debian/changelog]: righto - I should fix my script

Comment (domcleal) [inline plugins/ruby-puppetdb-foreman/debian/changelog]: There's one in scripts/ now.

**Comment by domcleal** [comment]
This is for both nightly and 1.10 repos I guess, because of the 1.10 fix?

**Comment by dLobatog** [comment]
@domcleal Yup, it's only relevant to those versions.

**Comment by domcleal** [comment]
Merged as 637a503, a8fe000 and building - thanks @dLobatog.

**Pull request closed without merging**